enterprise/server/raft/constants/constants.go

```diff
@@ -70,6 +75,9 @@ var (
 	// The last rangeID that was generated.
 	LastRangeIDKey = keys.MakeKey(SystemPrefix, []byte("last_range_id"))
 
+	// A prefix to prepend to transaction records.
+	TxnRecordPrefix = keys.MakeKey(SystemPrefix, []byte("txn-record-"))
```

Comment:
nit: is it worth specifying that it's a coordinator record to distinguish from replica written value?